docs/guides/setup_concord_with_env.md

```diff
@@ -0,0 +1,45 @@
+# Setting up with env variables
+
+## What is an "env variable"?
+
+Env variables are values that are dynamic-named that can affect how processes behave in case their values are edited. It can store values like paths, names, ints, and anything (except for files). Widely used in many applications.
```

Comment:
I'd probably say "Environment variables are dynamically named values that can affect how processes behave when their values are edited. The shell is responsible for defining and managing environment variables. They can store values like paths, names, ints, and anything (except for files). They are widely used in many applications."